### PR TITLE
Bugfix/117949

### PR DESCRIPTION
**Pull request by nikzayn**
#### What type of PR is this?
/kind bug
/kind documentation

#### What this PR does / why we need it:
This PR actually replaces the references of this URL [https://storage.googleapis.com/kubernetes-release]https://storage.googleapis.com/kubernetes-release to [https://dl.k8s.io]https://dl.k8s.io and also including references related from this URL [gs://kubernetes-release]gs://kubernetes-release to this [https://dl.k8s.io]https://dl.k8s.io 

#### Which issue(s) this PR fixes:
117949[https://github.com/kubernetes/kubernetes/issues/117949]

#### Special notes for your reviewer:
Just recheck the links once before merging!

#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.2.md [hunk 1]
```diff
@@ -58,27 +58,27 @@
 
 filename | sha256 hash
 -------- | -----------
-[kubernetes.tar.gz](https://storage.googleapis.com/kubernetes-release/release/v1.2.7/kubernetes.tar.gz) | `53db157923c17fa7a0addb3e4dfe7d1b9194b9266a87d371a251d5bb790a1832`
-[kubernetes-src.tar.gz](https://storage.googleapis.com/kubernetes-release/release/v1.2.7/kubernetes-src.tar.gz) | `e6e46831706743d8263581d0575507cf5ffc265096d22e5e84cf1c3ae925db5e`
+[kubernetes.tar.gz](https://dl.k8s.io//release/v1.2.7/kubernetes.tar.gz) | `53db157923c17fa7a0addb3e4dfe7d1b9194b9266a87d371a251d5bb790a1832`
```
Comment (Namanl2001) [inline CHANGELOG/CHANGELOG-1.2.md:61]:
```suggestion
[kubernetes.tar.gz](https://dl.k8s.io/release/v1.2.7/kubernetes.tar.gz) | `53db157923c17fa7a0addb3e4dfe7d1b9194b9266a87d371a251d5bb790a1832`
```

applies for every change

**Comment by Namanl2001** [comment]
@nikzayn @rjsadow should this PR be closed for #117950?

**Comment by rjsadow** [comment]
> @nikzayn @rjsadow should this PR be closed for #117950?

Yes.

**Pull request closed without merging**